docs/upgrading-stack.asciidoc

```diff
@@ -45,6 +45,7 @@ and make the necessary changes so your code is compatible with {version}:
 +
 --
 ** {beats-ref}/breaking-changes.html[Beats {version} breaking changes]
+** {apm-server-ref}/upgrading.html#breaking-changes[APM Server {version} breaking changes]
```

Comment:
with your latest changes, the breaking changes will become a separate topic rather than a sub-section, so you need to use:

```
{apm-server-ref}/breaking-changes.html[APM Server {version} breaking changes]

```